backend/compact-connect/lambdas/python/provider-data-v1/tests/function/test_handlers/test_provider_users.py

```diff
@@ -194,7 +194,12 @@ def test_post_provider_military_affiliation_sets_previous_record_status_to_inact
 
         event = self._when_testing_post_provider_user_military_affiliation_event_with_custom_claims()
 
-        resp = provider_user_me_military_affiliation(event, self.mock_context)
+        # We'll set the 'current datetime' to a date after the initial military affiliation record
```

Comment:
after the record expires? is uploaded?

```suggestion
        # We'll set the 'current datetime' to a date after the initial military affiliation record expires/was uploaded
```